tests/functional/s3cmd/tests.js

```diff
@@ -283,7 +304,11 @@ describe('s3cmd put and get bucket ACLs', function aclBuck() {
         exec(['setacl', `s3://${bucket}`, '--acl-public'], done);
     });
 
-    it('should get canned ACL that was set', done => {
+    // TODO: Unskip when getBucketLocation has been implemented
+    // We currently return NotImplemented to ?location queries, so `s3cmd info`
+    // will not print info for canned ACL (but once we support those queries,
+    // this test should work again)
+    it.skip('should get canned ACL that was set', done => {
```

Comment:
please make sure we have a functional test of this behavior at least with another client.